k8s/kube-prometheus/kube-prom-no-req-no-lim.jsonnet

```diff
@@ -0,0 +1,57 @@
+// Takes inspiration from
+// https://github.com/prometheus-operator/kube-prometheus/blob/c5360561fa3cb52297b652c263794ca474a0ab73/jsonnet/kube-prometheus/components/prometheus-operator.libsonnet
+{
+  values+:: {
+    alertmanager+: {
+      resources+: {
+        limits: {},
+        requests: { cpu: '1m', memory: '10Mi' },
+      },
+    },
+
+    blackboxExporter+: {
+      resources+: {
+        limits: {},
+        requests: { cpu: '1m', memory: '10Mi' },
+      },
+    },
+
+    grafana+: {
+      resources+: {
+        limits: {},
+        requests: { cpu: '1m', memory: '10Mi' },
```

Comment:
Note: these are very small values  (approaching zero), telling k8s that the resource footprint is very small, so taht these pods can all end up on the same node, even if that node itself has "only" two CPUs and little mem. This here is supposed to apply only during CI/local dev.